keras_cv/losses/iou_loss.py

```diff
@@ -55,14 +60,15 @@ class IoULoss(tf.keras.losses.Loss):
     ```
     """
 
-    def __init__(self, bounding_box_format, mode="log", **kwargs):
+    def __init__(self, bounding_box_format, mode="log", axis=None, **kwargs):
```

Comment:
can axis come with a default value?

Reply:
Hmm, I left `None` as default because it would mean no mean at all as we would use it in YoloX

Alternatively, I can add `axis=-1` as the default value maybe? The output of this loss, in that case, would be (batch_size,) if reduction="none"

Reply:
Hmmm...then why do we need to support a `axis` argument here, if YOLOX uses `None`? (as other keras losses don't have such argument)

Reply:
In case other models use the loss in the future.